core-compose/src/androidTest/java/com/squareup/workflow/ui/compose/CompositionRootTest.kt

```diff
@@ -49,7 +47,10 @@ class CompositionRootTest {
       }
     }
 
-    findByText("one\ntwo").assertIsDisplayed()
+    // These semantics used to merge, but as of dev15, they don't, which seems to be a bug.
+    // https://issuetracker.google.com/issues/161979921
+    onNodeWithText("one", useUnmergedTree = true).assertIsDisplayed()
+    onNodeWithText("two", useUnmergedTree = true).assertIsDisplayed()
```

Comment:
I tried removing this `useUnmergedTree` argument locally, and the test still passed.